### PR TITLE
[Regression Fix] Readd loading icon

### PR DESCRIPTION
**Pull request by Mikescops**
Fix for regression added by #468.

With the last vue-virtual-grid update we can set a loader component until the next batch is loaded.

### CODE REVIEW

File: src/components/Loader.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (skjnldsv) [inline src/components/Loader.vue:1]: copyright header ;)

**Review by skjnldsv — APPROVED** [review]
Minor change request, good otherwise

File: src/components/Loader.vue (hunk 1 above)
Comment (Mikescops) [reply src/components/Loader.vue:20]: Is that really needed? I added it, but one LICENSE at the root of the project looks enough to me.

Comment (skjnldsv) [reply src/components/Loader.vue:20]: Well, that's how we do things here :)

File: src/components/Loader.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!--
+ - @copyright Copyright (c) 2020 Nextcloud
+ -
+ - @license GNU AGPL version 3 or any later version
```
Comment (skjnldsv) [inline src/components/Loader.vue:4]: Nope, you're the owner!

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: src/components/Loader.vue (hunk 2 above)
Comment (skjnldsv) [reply src/components/Loader.vue:4]: Btw please also add yourself as the author for the other files you edited

Comment (Mikescops) [reply src/components/Loader.vue:4]: :disappointed:

Comment (Mikescops) [reply src/components/Loader.vue]: done

**Review by skjnldsv — APPROVED** [review]

**Comment by Mikescops** [comment]
@skjnldsv it was not a merge on master...

Need to review #478 first

**Comment by skjnldsv** [comment]
I thought it was on purpose so I could review 478 after :grin:

**Pull request merged**